src/act.obj.cpp

```diff
@@ -2400,7 +2404,7 @@ void perform_wear(struct char_data * ch, struct obj_data * obj, int where)
                               "YOU SHOULD NEVER SEE THIS MESSAGE.  PLEASE REPORT.\r\n",
                               "You already have something on each of your ankles.\r\n",
                               "You are already wearing something on your feet.\r\n",
-                              "You're already wearing something on your feet.\r\n"
+                              "You already have something in your mouth.\r\n",
```

Comment:
You've replaced the wear-feet message with the mouth one, so people with shoes on who try to wear more shoes will wonder why they're trying to eat their shoes. Recommend restoring `"You're already wearing something on your feet.\r\n",` before this.

Reply:
Other than this, it looks good!

Reply:
Fixed the dangling comma while I was at it.